HackYourFuture-CPH/Guide-IT
# Question page

# Description

This PR is to submit quiz answer ids and user ids to the quiz-results table in the database when a user clicks next. Show questions from the database, and fetch answers as objects on the questions array.

Problems still to be solved:
Handle authenticated users
on See Results button from last question, navigating to quiz-results page.

Fixes # https://github.com/HackYourFuture-CPH/Guide-IT/issues/137

# How to test?

Run on seeded database and check that questions and answers are being fetched successfully. Check that answers are being submitted successfully.

# Checklist
